src/appleseed.studio/mainwindow/mainwindow.cpp

```diff
@@ -388,9 +388,22 @@ void MainWindow::build_menus()
     m_ui->menu_view->addSeparator();
 
     QAction* fullscreen_action = m_ui->menu_view->addAction("Fullscreen");
+    fullscreen_action->setCheckable(true);
     fullscreen_action->setShortcut(Qt::Key_F11);
+    fullscreen_action->setObjectName("Fullscreen");
```

Comment:
The name should be `action_fullscreen` in order to be consistent with other names.

Reply:
Sounds good. I've pushed a commit that addresses this.